toolkit/themes/shared/design-system/config.js

```diff
@@ -37,6 +37,8 @@ const MEDIA_QUERY_PROPERTY_MAP = {
   "prefers-contrast": "prefersContrast",
 };
 
+const BASE_SELECTOR = ":root,\n" + ":host(.anonymous-content-host) {\n";
```

Comment:
Moving this to a template string could be a smaller change that helps clarity a bit, but not too concerned

```suggestion
const BASE_SELECTOR = `:root,
:host(.anonymous-content-host) {
`;
```

Reply:
I just find the weird indentations needed to make template strings work kind of hard to read. I can see how other people would find all the "\n" and adding of strings hard to read though 😆

I'm going to leave this as is but let's revisit making the file consistent one way or another when we're moving all this to central.